src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Web/Areas/VcsAdmin/Pages/ManageOrganisations.cshtml

```diff
@@ -18,10 +18,12 @@
 
 <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--l">
 
-
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-one-third">
         <form method="post">
+            <button start-hidden="" class="govuk-button govuk-button--secondary fh-open-close-button " data-open-close-mobile="filters-component" data-open-close-mobile-hide="Hide filter" data-open-close-mobile-default="hide" data-module="govuk-button" type="button">
```

Comment:
some other areas of the app use a <open-close-button>. Does it matter much or is this more specific to mobile?

Reply:
I just took this code directly from the [manage] "Services" page in Manage and it worked

Reply:
It's likely it's not been updated since the `open-close-button` helper was added to the shared kernel. If we've got a shared component we should probably use it?

Reply:
I can see the custom button in the shared razor. It does the same thing by the looks of it. I'll leave it up to you. Feel free to close this comment if your not changing.

Reply:
Not sure why, but replacing this with the shared component doesn't work. It has a static "Show filters" text on the screen in both desktop and mobile doesn't show a button. I'll just leave it like this as-is for the time being

Reply:
Nvm managed it (thanks @Top-Cat) [8673f3b](https://github.com/DFE-Digital/fh-services/pull/278/commits/8673f3be9e3fb064b1ff01969c49b088b79f7d4f)